api/tests/unit/organisations/test_unit_organisations_tasks.py

```diff
@@ -649,3 +968,20 @@ def test_unrestrict_after_api_limit_grace_period_is_stale(
     assert organisation4.stop_serving_flags is True
     assert organisation4.block_access_to_admin is True
     assert getattr(organisation4, "api_limit_access_block", None) is None
+
+
+def test_recurring_tasks(mocker: MockerFixture, settings: SettingsWrapper) -> None:
+    # Given
+    settings.ENABLE_API_USAGE_ALERTING = True
+    register_task_mock = mocker.patch("organisations.tasks.register_recurring_task")
+
+    # When
+    register_recurring_tasks()
+
+    # Then
+    register_task_mock.call_args_list == [
+        call(run_every=timedelta(seconds=43200)),
+        call(run_every=timedelta(seconds=1800)),
+        call(run_every=timedelta(seconds=43200)),
+        call(run_every=timedelta(seconds=43200)),
+    ]
```

Comment:
I have a feeling you're going to hate me for this, because I know you only really implemented this test for the coverage... but this test feels fairly pointless without actually asserting the functions that we're wrapping. How tricky is it to extend this test to check the task functions? 

I think we'd just need to do something like: 

```python
assert register_task_mock.return_value.call_args_list = [
    call(task_function_1),
    ...,
    call(task_function_n),
]
```

Reply:
Yeah good call. I considered doing this before but I (incorrectly) thought the mocking would be more complicated.

Added.